session_3/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @articles.each do |article| %>
+      <tr>
+        <td><%= article.name %></td>
+        <td><%= article.content %></td>
+        <td><%= link_to 'Show', article %></td>
```

Comment:
Since you have not enclosed this in `if can? :show, article` - wouldn't this be visible to all users, regardless it is a public or a private article?

Reply:
Ohh yeah.... Am on it

Reply:
I have done the necessary, I think
Also I have completed the extra tasks.